locale/ar/docs/guides/debugging-getting-started.md

```diff
@@ -0,0 +1,213 @@
+---
+title: Debugging - Getting Started
```

Comment:
translate title

Reply:
@Aissaoui-Ahmed Roger that :)

Reply:
@Aissaoui-Ahmed requested changes has been made to the file... take a look